.github/workflows/build.yml

```diff
@@ -172,7 +174,6 @@ jobs:
           key: ${{ steps.compute_lockfile_hash.outputs.hash }}
 
       - name: Install dependencies
-        if: steps.cache_dependencies.outputs.cache-hit == '' || needs.job_get_metadata.outputs.force_skip_cache == 'true'
```

Comment:
Actually I don't think we need this, when we hit the cache this step will be more or less instantenous anyhow. May help here or there with an inconsistency.